src/pages/graphql/schema/checkout/queries/get-payment-sdk.md

```diff
@@ -44,33 +44,60 @@ query {
+                        }
+                    ]
+                },
+                {
+                    "code": "payment_services_paypal_apple_pay",
+                    "params": [
+                        {
+                            "name": "src",
+                            "value": "https:\/\/www.paypal.com\/sdk\/js?client-id=AeFUI2PCqy7MicQ1gLbJf_8T7XUlEG51Pdd7sn2LE3d0_qAVhVBm4VOpPGmV08S9qsMCPXSCiBslhV53&intent=capture&locale=en_US&merchant-id=2PZE7VHLY4B94&currency=USD&commit=false&disable-funding=bancontact,boleto,boletobancario,blik,eps,giropay,ideal,itau,mercadopago,multibanco,mybank,oxxo,payu,p24,satispay,sepa,sofort,trustly,wechatpay,credit,card&enable-funding=applepay&components=buttons"
+                        }
+                    ]
+                },
+                {
+                    "code": "payment_services_paypal_google_pay",
+                    "params": [
+                        {
+                            "name": "src",
+                            "value": "https:\/\/www.paypal.com\/sdk\/js?client-id=AeFUI2PCqy7MicQ1gLbJf_8T7XUlEG51Pdd7sn2LE3d0_qAVhVBm4VOpPGmV08S9qsMCPXSCiBslhV53&intent=authorize&locale=en_US&merchant-id=2PZE7VHLY4B94&currency=USD&disable-funding=bancontact,boleto,boletobancario,blik,eps,giropay,ideal,itau,mercadopago,multibanco,mybank,oxxo,payu,p24,satispay,sepa,sofort,trustly,wechatpay&components=googlepay"
+                        }
+                    ]
+                },
+                {
+                    "code": "payment_services_paypal_hosted_fields",
+                    "params": [
+                        {
+                            "name": "src",
+                            "value": "https:\/\/www.paypal.com\/sdk\/js?client-id=AeFUI2PCqy7MicQ1gLbJf_8T7XUlEG51Pdd7sn2LE3d0_qAVhVBm4VOpPGmV08S9qsMCPXSCiBslhV53&intent=capture&locale=en_US&merchant-id=2PZE7VHLY4B94&currency=USD&disable-funding=bancontact,boleto,boletobancario,blik,eps,giropay,ideal,itau,mercadopago,multibanco,mybank,oxxo,payu,p24,satispay,sepa,sofort,trustly,wechatpay&components=hosted-fields"
+                        },
+                        {
+                            "name": "data-partner-attribution-id",
+                            "value": "MagentoPayments_SP_PCP_Int"
+                        },
+                        {
+                            "name": "data-client-token",
+                            "value": "eyJicmFpbnRyZWUiOnsiYXV0aG9yaXphdGlvbkZpbmdlcnByaW50IjoiMTFiZmFjZGM5YWEyM2ZhZjdmNTQwMzc0NGZmYzEwMjI4YjFjODBmZjg4NDdlYjcyMjMyMmM1OTE0MTU3OWYzZHxtZXJjaGFudF9pZD1yd3dua3FnMnhnNTZobTJuJnB1YmxpY19rZXk9NjNrdm4zN3Z0MjlxYjRkZiZjcmVhdGVkX2F0PTIwMjQtMDItMTNUMTU6Mjc6NTkuNDg5WiIsInZlcnNpb24iOiIzLXBheXBhbCJ9LCJwYXlwYWwiOnsiaWRUb2tlbiI6bnVsbCwiYWNjZXNzVG9rZW4iOiJBMjFBQUlEc1hFbHFlVzh6d1FTTGZsVFdkMHR1UkhIYlBYVTdfYTQzNVFxUVQ5MTRmUjhzclN1RTQzdlg4TnVXV0N1NHZIeUVoVG1BUnhVekdnU3R3VDJCNVFtczBrVjRnIn19"
```

Comment:
is this a valid token?

Reply:
I took it from the example wiki where we had all valid examples